pandas/tests/groupby/test_apply.py

```diff
@@ -363,11 +363,19 @@ def f(group):
         tm.assert_frame_equal(result.loc[key], f(group))
 
 
-def test_apply_chunk_view():
+@pytest.mark.parametrize("as_index", [False, True])
```

Comment:
Do you even need this in the test? Looks like the test itself just remove the `True` case